src/platforms/posix/drivers/df_lsm9ds1_wrapper/df_lsm9ds1_wrapper.cpp

```diff
@@ -572,26 +570,22 @@ int DfLsm9ds1Wrapper::_publish(struct imu_sensor_data &data)
 
 	math::Vector<3> vec_integrated_unused;
 	uint64_t integral_dt_unused;
-
+	// apply sensor rotation on the accel measurement
+	rotate_3f(_rotation, data.accel_m_s2_x, data.accel_m_s2_y, data.accel_m_s2_z);
```

Comment:
I see how this can be more efficient looking at rotate_3f. I would prefer if it's implemented in the same way for all drivers, so this change is not consistent with the others as far as I can tell.